src/definitions/modules/video.js

```diff
@@ -85,20 +93,218 @@ $.fn.video = function(parameters) {
         },
 
         create: function() {
-          var
-            image = $module.data(metadata.image),
-            html = templates.video(image)
-          ;
-          $module.html(html);
-          module.refresh();
-          if(!image) {
-            module.play();
-          }
-          module.debug('Creating html for video element', html);
+          $video = element
+          module.debug('Reefrencing html for video element');
         },
-
+        
+        
+        control: {
+          
+          playpause: function($playpause) {
+            $playpause = $($playpause);
+            // from UI to video
+            $playpause.on('click' + eventNamespace, function() {
+              if(element.paused) {
+                element.play();
+              } else {
+                element.pause();
+              }
+            });
+            // from video to UI
+            $module
+              .on('play' + eventNamespace, function() {
+                $playpause.addClass('active')
+              }).on('pause' + eventNamespace, function() {
+                $playpause.removeClass('active')
+              });
+          },
+          
+          // see https://developer.mozilla.org/fr/docs/Web/API/HTMLMediaElement#playbackRate
+          // mode from 'switch', 'push'
+          playbackrate: function($playbackrate, rate, mode) {
+            $playbackrate = $($playbackrate);
+            // from UI to video
+            switch(mode) {
+              case 'push': default:
+                $playbackrate.on('mousedown' + eventNamespace, function() {
+                  element.playbackRate = rate;
+                });
+                $playbackrate.on('mouseup' + eventNamespace + ' mouseleave' + eventNamespace, function() {
```

Comment:
About this example with ``module.change.playbackRate``: how is the best to configure the 2 buttons (forward and backward) that would both call the same function, but with a different rate (say 6 and -4) ? Should this setting be inside the module or in a ``data-playback-rate``attribute for each of them ?